### ISSUE TITLE
Merge PR #2 into code without breaking it 

### ISSUE THREAD

**Issue by sbryngelson** [comp-physics/RBC3D#12]
Merge PR #2  into code without breaking it (the new code or the old code)

**Comment by suzanmanasreh** [comment]
I don't think this is a good title or description because nothing in the new module can break anything.

**Comment by sbryngelson** [comment]
That's fair enough, and I'm open to other titles or descriptions. The point is that I would like to see an example of a case that uses this feature of the additional code and merges the additional code into `common/`. Does that make sense? I haven't looked at all the details in a while, so let me know.
